MapboxNavigation/NavigationMapView.swift

```diff
@@ -22,6 +22,12 @@ let sourceOptions: [MGLShapeSourceOption: Any] = [.maximumZoomLevel: 16]
 @objc(MBNavigationMapView)
 open class NavigationMapView: MGLMapView {
     
+    //MARK: Class Constants
+    public static let defaultAltitude = 1000.0
```

Comment:
Explicitly specify CLLocationDistance as the type. That along with a documentation comment will make it clear how the developer is supposed to interpret this option. (CLLocationDistance is defined to be measured in meters.)